### PR TITLE
Example on how to handle translation of Javascript strings (backport

### LINKED ISSUE #1342: Use the new translator to translate Javascript-save strings
In the old translation system, the smarty function `l` took an optional parameter `js=1` to output a string that could be used in Javascript. 
https://devdocs.prestashop.com/1.7/development/internationalization/translation/using-the-translator/#smarty-templates does not mention how to handle this case with the new translation system.

### PR DESCRIPTION
**Pull request by lmeyer1**
| Questions     | Answers
| ------------- | -------------------------------------------------------
| Branch?       | 8.x
| Description?  | Give an example on how to handle translation of Javascript strings in smarty templates.
| Fixed ticket? | Fixes #1342, backport of PR #1344

### CODE REVIEW

**Review by matks — APPROVED** [review]

**Comment by matks** [comment]
Thank you @lmeyer1

File: development/internationalization/translation/using-the-translator.md [hunk 1]
```diff
@@ -106,6 +106,13 @@ If you have have replacements to peform in your wording, then there are two opti
     <div>{l s='There are %products_count% items in your cart.' sprintf=['%products_count%' => $cart.products_count] d='Shop.Theme.Checkout'}</div>
     ```
 
+**Note:** The `l` macro does not support escaping the strings for javascript. Instead you can assign the translation to a variable and escape it afterwards:
+
+```html
```
Comment (SharakPL) [inline development/internationalization/translation/using-the-translator.md:111]: Try `smarty` instead of `html`

Comment (lmeyer1) [reply development/internationalization/translation/using-the-translator.md:111]: @SharakPL This PR was already merged, when you reviewed it. It seems to be useless to review closed PRs.

Also, `html` is used elsewhere in devdocs. Why would we change just for this code snippet ?

Comment (SharakPL) [reply development/internationalization/translation/using-the-translator.md:111]: Sorry, didn't notice it's merged already. Afaik markdown codeblock supports smarty syntax and it's .tpl file, not .html so I simply suggest using correct syntax.

**Pull request merged**